packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -2788,14 +2788,10 @@ Generated by [AVA](https://avajs.dev).
       CACHE_DIR: '.netlify/cache',␊
       IS_LOCAL: true,␊
       NETLIFY_BUILD_VERSION: '1.0.0',␊
-      SITE_ID: undefined,␊
```

Comment:
JSON serialisation contains undefined values in comparison of `v8.serialise`. which should not make any difference just the snapshot looks differently.

Reply:
I think you mean the other way around, i.e. JSON serialisation omits `undefined` values?

Reply:
yep :D